src/components/RuleCard/index.jsx

```diff
@@ -712,33 +714,53 @@ function RuleCard({
       </CardContent>
       {!readOnly && (
         <CardActions className={classes.cardActions}>
-          <Link
-            className={classes.link}
-            to={{
-              pathname: '/rules/create',
-              state: {
-                rule,
-              },
-            }}>
-            <Button color="secondary">Duplicate</Button>
-          </Link>
-          <Link
-            className={classes.link}
-            to={
-              rule.rule_id
-                ? `/rules/${rule.rule_id}`
-                : `/rules/create/${rule.scheduledChange.sc_id}`
-            }>
-            <Button color="secondary">Update</Button>
-          </Link>
-          <Button color="secondary" onClick={() => onRuleDelete(rule)}>
+          {user ? (
+            <Link
+              className={classes.link}
+              to={{
+                pathname: '/rules/create',
+                state: {
+                  rule,
+                },
+              }}>
+              <Button color="secondary">Duplicate</Button>
+            </Link>
+          ) : (
+            <Button color="secondary" disabled>
+              Duplicate
+            </Button>
+          )}
+          {user ? (
+            <Link
+              className={classes.link}
+              to={
+                rule.rule_id
+                  ? `/rules/${rule.rule_id}`
+                  : `/rules/create/${rule.scheduledChange.sc_id}`
+              }>
+              <Button color="secondary">Update</Button>
+            </Link>
+          ) : (
+            <Button color="secondary" disabled>
+              Update
+            </Button>
+          )}
+          <Button
+            color="secondary"
+            disabled={!user}
+            onClick={() => onRuleDelete(rule)}>
             Delete
           </Button>
-          {requiresSignoff &&
+          {!readOnly &&
```

Comment:
I just realized this check is unnecessary since we do it on top before rendering the card actions.